### ISSUE TITLE
ttnn.logaddexp_bw operation fails with low PCC when tensors are in mixed precisions

### ISSUE THREAD

**Issue by nemanjagrujic** [tenstorrent/tt-metal#13930]
ttnn.logaddexp_bw operation fails with low PCC when tensors are in mixed precisions (eg one tensor is blofat16 and other is bfloat8_b).

Steps to reproduce

You can reproduce this error with sweep test `tests/sweep_framework/sweeps/eltwise/unary_backward/logaddexp_bw.py`. To run:

```
python3 tests/sweep_framework/runner.py --elastic cloud --module-name eltwise.unary_backward.logaddexp_bw 
```

Results can be found on elastic cloud as explained here: `https://github.com/tenstorrent/tt-metal/tree/main/tests/sweep_framework`

**Comment by prajaramanTT** [comment]
@nemanjagrujic is this still an open issue ? If not, can you please close this ? Thanks.

**Comment by nemanjagrujic** [comment]
@prajaramanTT Tested, this is still an issue.

**Comment by KalaivaniMCW** [comment]
@mcw-vishal Pls verify if this has the same issue as https://github.com/tenstorrent/tt-metal/issues/13973 and update

**Comment by mcw-vishal** [comment]
Yes, Since recip and exp are involved, The behaviour is same as https://github.com/tenstorrent/tt-metal/issues/13973. I will add the necessary dtype checks.

**Comment by mcw-vishal** [comment]
Merged https://github.com/tenstorrent/tt-metal/pull/24271 to error out unsupported dtypes
